src/components/App.js

```diff
@@ -157,7 +162,11 @@ class App extends Component<{}, State> {
   getFolderArray = (): Array<any> => {
     const folders = []
     this.state.notes.forEach(note => {
-      if (note.folder !== '') {
+      if (
+        note.folder !== '' &&
+        note.folder !== 'archive' &&
+        note.folder !== 'all notes'
+      ) {
```

Comment:
It might be time to wrap this logic in a function. Maybe end with a conditional something like "if (!isEmptyFolder(note.folder) && !isSystemFolder(note.folder))"?